content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -0,0 +1,346 @@
+---
+reviewers:
+- hasheddan
+- pjbgf
+- saschagrunert
+title: Seccomp
```

Comment:
```suggestion
title: Isolate a workload using Linux Seccomp
```
?

Reply:
This was to match the existing `AppArmor` title.